### DISCUSSION TITLE
Unrecognisable author of contributions

### DISCUSSION THREAD

**Discussion by plkpiotr** [community/community#22920] [New to GitHub]
<p>I’ve just noticed that my contributions from holidays and previous cease to be assigned to my GitHub profile (for example in these repositories: <a href="https://github.com/plkpiotr/forum/commits/master" rel="nofollow noopener">forum </a>or <a href="https://github.com/plkpiotr/squash-court/commits/master" rel="nofollow noopener">squash-court</a>)<br>
At this time I should have over 500 (like a couple days ago) but I see only 118.<br>
Does anyone have the same? It is the work of brave-new-changes?</p>

**Comment by angela-crist** [comment]
<p>Hi <a class="mention" href="/plkpiotr">@plkpiotr</a>,</p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our Community organized so users can more easily find information.</p>
<p>As you’ll notice, your Topic is now in the How to use Git and GitHub board. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Cheers!</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>In taking a look at plkpiotr/forum I see commits that look like this:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/1/1c331f3dc1403c9d28bc9eebb148d6d7e53a9735.png" data-download-href="/uploads/short-url/41sTEuhLLYB3r4wVSX8MiVoOESN.png?dl=1" title="plkpiotr.png"><img src="https://user-images.githubusercontent.com/1038121/181073711-c1dabdfd-9ba7-4ad3-a298-d3fe8349ab25.png" alt="plkpiotr.png" data-base62-sha1="41sTEuhLLYB3r4wVSX8MiVoOESN" width="690" height="113" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/1/1c331f3dc1403c9d28bc9eebb148d6d7e53a9735_2_690x113.png, https://aws1.discourse-cdn.com/github/optimized/2X/1/1c331f3dc1403c9d28bc9eebb148d6d7e53a9735_2_1035x169.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/1/1c331f3dc1403c9d28bc9eebb148d6d7e53a9735_2_1380x226.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/1c331f3dc1403c9d28bc9eebb148d6d7e53a9735_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">plkpiotr.png</span><span class="informations">2398×394 95.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>When I clone the repository locally, I see this:</p>
<pre><code>commit b76d80baf57037dc43953d37bfcc0bffd00688a2
Author: Piotr Pałka &lt;redacted@gmail.com&gt;
Date: Mon Sep 25 17:05:31 2017 +0200

    Update README.md

commit 58fad2ce866a56d6d5e85971287ede7c173f1222
Author: plkpiotr &lt;redacted@redacted.pl&gt;
Date: Mon Sep 25 13:02:37 2017 +0200

    Refactoring
</code></pre>
<p>Notice that the top one is using the <a href="http://gmail.com">gmail.com</a> email address (which is apparently associated with your GitHub account) and the bottom one is using a different email address (which is not associated with your GitHub account). You can associate the second email address with your GitHub account to get credit for those commits as well.</p>
<p>See the recent Support Pro-Tips article <a href="https://github.community/t/why-is-my-commit-associated-with-the-wrong-person/10199">Why is my commit associated with the wrong person?</a> for more on how GitHub gives credit for commits.</p>
<p>I hope that helps!</p>

**Comment by plkpiotr** [comment]
<p>Thank you so much!</p>
<p>I forgot that I deleted my e-mail a few days ago :slight_smile:<br>
I added old e-mail address anew and everything is okay! :wink:</p>

**Comment by slee5777** [comment]
<p>I used command line to submit a pull request.  I entered my GitHub user name but failed to provide email address.  It was mergered to the Master.  Any chance to update the AUTHOR_EMAIL only without using “rebase”?  </p>
<p>Sorry, I was not aware of the no-reply github email at that time.  Now, I set it up properly.</p>

**Comment by nadiajoyce** [comment]
<p>Hi <a class="mention" href="/slee5777">@slee5777</a>,</p>
<p>You can follow the steps here to update the author info:</p>
<p><a href="https://help.github.com/articles/changing-author-info/" rel="nofollow noopener">https://help.github.com/articles/changing-author-info/</a></p>
<p>Cheers!</p>
